test/JDBC/upgrade/latest/schedule

```diff
@@ -400,6 +400,7 @@ sys-systypes
 openjson
 BABEL-3702
 BABEL-3696
+BABEL-3697
```

Comment:
latest is only for MVU with no version change, please add to a schedule file which will exercise 3.1 to 3.2

Reply:
The json_modify function was originally implemented in 14_6 and the file testing it (babel-937) is in the upgrade schedule for 14_6, 14_7, 14_8, 15_1, 15_2, and 15_latest which tests that the function works properly still in older versions.

Reply:
this is not about function working in old versions. The new test cases added in Babel-3697 will only run on latest server (new create). However, another test case is to run them after upgrade. So, tests cases in Babel-3697 are not running for upgrade use case. Whenever we change an upgrade file, it is good practice to run corresponding tests both in fresh create and after upgrade.